js_machine_front/src/scenes/digestView/styles/leftArrowButton.css

```diff
@@ -1,17 +1,17 @@
-.LeftArrowButton {
+.digestview__leftArrowButton {
     position: fixed;
     top: 7%;
     left: 3%;
     cursor: pointer;
 }
 
-.LeftArrowButton-text {
+.digestview__leftArrowButton-text {
```

Comment:
if you define styles for element inside another element you should write like left-arrow-button__text,
block-element-element is forbidden